Security/Handler/CacheSecurityHandler.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Security\Handler;
+
+use Sonata\AdminBundle\Admin\AdminInterface;
+
+class CacheSecurityHandler implements SecurityHandlerInterface
+{
+    /**
+     * @var SecurityHandlerInterface
+     */
+    private $handler;
+
+    /**
+     * @var array
+     */
+    private $cache;
+
+    /**
+     * @param SecurityHandlerInterface $handler
+     */
```

Comment:
Other constructors have param comments on constructors. IMHO, should be kept for consistency.